app/main.py

```diff
@@ -1 +1,41 @@
-#  write your code here
+class Animal:
+    def __init__(self,
+                 name: str,
```

Comment:
Pass the paramters to the nextline https://stackoverflow.com/a/41591509

Reply:
@Kroll410 Pushed the update 👍